### PR TITLE
[stable/percona-xtradb-cluster] Make metrics exporter working out of the box

### PR DESCRIPTION
**Pull request by harloprillar**
Signed-off-by: Alexander Scherbakov <harloprillar@gmail.com>

#### Is this a new chart
No

#### What this PR does / why we need it:
Current chart version contain ability to enable metrics exporter, but if you try to do it, cluster will newer up because of metrics exporter container not configured in chart and pod shedule will crash.
This PR configure metrics exporter making them work out of the box after enabling in values.yml.

#### Which issue this PR fixes
None

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ x] Chart Version bumped
- [x ] Variables are documented in the README.md
- [x ] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by Basster** [comment]
can we have this merged, please! Works like charm!

**Review by Basster — COMMENTED** [review]
works for me, like a charm.

File: stable/percona-xtradb-cluster/files/functions.sh [hunk 1]
```diff
@@ -96,10 +96,17 @@ EOSQL
             if [ "$MYSQL_DATABASE" ]; then
                 echo "GRANT ALL ON \`"$MYSQL_DATABASE"\`.* TO '"$MYSQL_USER"'@'%' ;" | "${mysql[@]}"
             fi
+            echo 'FLUSH PRIVILEGES ;' | "${mysql[@]}"
+        fi            
 
+        if [ "$METRICS_EXPORTER_PASSWORD" ]; then
+            echo "CREATE USER 'exporter'@'localhost' IDENTIFIED BY '"$METRICS_EXPORTER_PASSWORD"' ;\
```
Comment (Basster) [inline stable/percona-xtradb-cluster/files/functions.sh:103]: I guess localhost should be replaced by `127.0.0.1`

**Comment by kvaps** [comment]
please reopen

**Comment by phal0r** [comment]
@kvaps 
You should give the percona kubernetes operator a try. I think it approaches database management better than a helm chart ever could.

**Comment by kvaps** [comment]
@phal0r thanks for the suggestion, but tight now I need exactly helm chart.
That's not a big problem to merge these changes locally, they are working fine, people are using it, so why to not get merged?

**Comment by phal0r** [comment]
@kvaps 
ok, sure :)
I think, that it will never get merged as this global helm repo is deprecated and all app providers have their own helm repos now. It seems, that Percona only focusses on the Operator: https://github.com/Percona-Lab/percona-helm-charts

**Pull request closed without merging**